src/librustc_lexer/src/cursor.rs

```diff
@@ -65,12 +67,8 @@ impl<'a> Cursor<'a> {
         self.initial_len - self.chars.as_str().len()
     }
 
-    /// Returns a `Chars` iterator over the remaining characters.
-    fn chars(&self) -> Chars<'a> {
-        self.chars.clone()
-    }
-
     /// Moves to the next character.
+    #[inline]
```

Comment:
Could you avoid adding `#[inline]` without profiling?

Reply:
I actually profiled it with a file with `#[cfg(False)] mod { /* 10.000 lines copied from other files */ }`. 
It barely made a difference, to small to measure in wall-time with criterion, but it was a slight improvement in instruction count with callgrind.

Reply:
Nevermind, I dropped the changes in `cursor.rs` entirely.